packages/tailwind-config/tailwind.config.mjs

```diff
@@ -95,12 +95,16 @@ export default {
       },
       zIndex: {
         1: '1',
+        nav: '50',
       },
       fontFamily: {
         sans: ['var(--font-aeonikpro)'],
       },
       spacing: {
         4.5: '18px',
+        5.5: '22px',
+        26: '104px',
+        50: '200px',
```

Comment:
I believe these could be inlined as probably it's only a single use-case so no need to pre-define it in tw config